### DISCUSSION TITLE
NVTDM x64

### DISCUSSION THREAD

**Discussion by vanquerubin** [leecher1337/ntvdmx64#191] [General]
Is there a way that I can open my 16 bit DOS App without going thru otvdm?

TIA

**Comment by leecher1337** [comment]
For DOS apps, just install NTVDMx64 and execute the DOS application. 
Forget about otvdm for DOS apps, it's not designed to work with DOS applications, otvdm is designed to run Win16 apps (Windows 3.x). You need NTVDMx64 for DOS apps.

**Reply by vanquerubin** [reply]
Is there a simple way of installing ntvdmx64 i am using accpac application.

**Reply by leecher1337** [reply]
1) Download autobuild from "Releases" and unpack it
2) Download required files to build directory as documented in https://github.com/leecher1337/ntvdmx64/blob/master/ntvdmpatch/doc/autobuild.txt  (Google is your friend)
3) Run autobuild-ccpu-fre.cmd and let it compile
4) Install from Releases-Directory according to the language of your system.

Edward Mendelson has some older ready-to-install builds available on his site, he may update them from time to time, but it's recommended to use autobuild procedure to ensure that you are using the latest version with latest bugfixes.
